.github/workflows/bump-formula-pr.yml

```diff
@@ -1,4 +1,4 @@
-name: bump-formula-pr
+ name: bump-formula-pr
```

Comment:
This has an extra space? I don't see it on the main branch:
https://github.com/grafana/alloy/blob/main/.github/workflows/bump-formula-pr.yml